extensions/amp-facebook/1.0/README.md

```diff
@@ -10,7 +10,7 @@ teaser:
 
```

Comment:
Also, CI is failing because there is a dead link in this file:

<img width="981" alt="Screen Shot 2021-10-01 at 09 11 27" src="https://user-images.githubusercontent.com/134745/135653289-791aefc3-d8f9-44ce-b864-fe86d07b36b6.png">

It looks like the links to `https://developers.facebook.com/docs/plugins/embedded-comments` need to be removed entirely because Facebook deprecated that “Embedded Comments Plugin” as seen in the Wayback Machine: https://web.archive.org/web/20210825175717/https://developers.facebook.com/docs/plugins/embedded-comments

![image](https://user-images.githubusercontent.com/134745/135653521-b007c074-cdd4-4234-912f-d13393614227.png)

Reply:
This seems a bug on the Facebook side because they still link to “Embedded Comments” in the nav menu for their Social Plugins: https://developers.facebook.com/docs/plugins

Reply:
Right, it's a 404 on their site. Anyways, I'll fix that in a follow up issue since this PR is mainly to fix the broken inline links to Preact/React components across various READMEs.

Reply:
Ah nevermind! Merging is blocked due to the broken links. Removed them.